packages/libs/react-components/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,64 @@
+import { IIconButtonProps } from '../IconButton';
+import { SystemIcons } from '../Icons';
+
+import {
+  StyledHeading,
+  StyledIconButton,
+  StyledNotification,
+  StyledText,
+} from './styles';
+
+import { VariantProps } from '@stitches/react';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcons)[keyof typeof SystemIcons];
+
+  title: string;
+  description?: string;
+  displayCloseButton?: boolean;
+  expand?: VariantProps<typeof StyledNotification>['expand'];
+
+  color?: VariantProps<typeof StyledNotification>['color'];
+
+  simple?: VariantProps<typeof StyledNotification>['simple'];
+
+  buttons?: React.ReactNode;
```

Comment:
Since this can accept any ReactNode it might be good to name it something more generic like footerContent